stream-tests/src/test/scala/org/apache/pekko/stream/scaladsl/FlowSwitchSpec.scala

```diff
@@ -0,0 +1,297 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+/*
+ * Copyright (C) 2014-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package org.apache.pekko.stream.scaladsl
+
+import org.apache.pekko.NotUsed
```

Comment:
one nit is that the project code style is to `import org.apache.pekko` and then the subsequent imports can begin `import pekko.stream...` (ie omit the org.apache bit).